src/server/rest/v1/service/BillingService.ts

```diff
@@ -314,8 +316,10 @@ export default class BillingService {
     if ((await Authorizations.canListUsers(req.user)).authorized) {
       userProject = [ 'userID', 'user.id', 'user.name', 'user.firstName', 'user.email' ];
     }
+    // Fetch tenant based on Tenant ID
+    const tenant = await TenantStorage.getTenant(req.user.tenantID);
```

Comment:
Do not get the tenant, it's already provided in the HTTP request in 'req.tenant'.